src/app/ui/editor/tool_loop_impl.cpp

```diff
@@ -530,10 +533,10 @@ class ToolLoopImpl : public ToolLoopBase,
         (m_layer->isTilemap() &&
          site.tilemapMode() == TilemapMode::Pixels &&
          site.tilesetMode() == TilesetMode::Manual &&
-         !m_ink->isSelection() ? ExpandCelCanvas::TilesetPreview:
-                                 ExpandCelCanvas::None) |
-        (m_ink->isSelection() ? ExpandCelCanvas::SelectionPreview:
-                                ExpandCelCanvas::None))));
+         !isSelectionPreview ? ExpandCelCanvas::TilesetPreview:
+                               ExpandCelCanvas::None) |
```

Comment:
Should we use parenthesis here?
```suggestion
         (!isSelectionPreview ? ExpandCelCanvas::TilesetPreview:
                                ExpandCelCanvas::None)) |
```
I know that the previous code didn't have them, but just wondering if it was wrong as well.

Reply:
Good to avoid confusion. Done.

Reply:
This has changed the condition and the result of the operation (go back to the previous version):
```
        (m_layer->isTilemap() &&
         site.tilemapMode() == TilemapMode::Pixels &&
         site.tilesetMode() == TilesetMode::Manual &&
         !isSelectionPreview ? ExpandCelCanvas::TilesetPreview:
                                 ExpandCelCanvas::None) |
```
is not equal to:
```
        (m_layer->isTilemap() &&
         site.tilemapMode() == TilemapMode::Pixels &&
         site.tilesetMode() == TilesetMode::Manual &&
         (!isSelectionPreview ? ExpandCelCanvas::TilesetPreview:
                                ExpandCelCanvas::None)) |
```
The first one returns `ExpandCelCanvas::TilesetPreview` or `ExpandCelCanvas::None` and the second one returns true or false (`1` or `0` in the `|` operation of ints).

Reply:
@Gasparoken I can take this PR and modify the commit directly if you wish (I'm planning to release a new version today if we can)